fv/attach_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```

Comment:
I tried initnially, but as you said, it is large as it is. So I felt it would be better to move it in a few file. I'll change the name to `bpf_attach_test.go` to make it more descriptive. Does it make sense?